docs/reference/analysis/tokenfilters/ngram-tokenfilter.asciidoc

```diff
@@ -185,6 +185,10 @@ Maximum length of characters in a gram. Defaults to `2`.
 (Optional, integer)
 Minimum length of characters in a gram. Defaults to `1`.
 
+`preserve_original`::
+(Optional, boolean)
+If set to true then it would also emit the original token. Defaults to `true`.
+
```

Comment:
The default is false? (And needs to be for backwards-compatibility reasons).

Reply:
@markharwood thanks for pointing this, Addressed it in my lastest comment.

Reply:
@markharwood I also rebased my feature branch with the latest master code and squashed my commits. Let me know if anything else required from my side.